Sources/PT.PM.Matching/PatternNormalizer.cs

```diff
@@ -97,19 +97,19 @@ public override PatternUst Visit(PatternIdRegexToken patternIdRegexToken)
             return new PatternIdRegexToken(regexString, patternIdRegexToken.TextSpan);
         }
 
-        public override PatternUst Visit(PatternIntRangeLiteral patternIntLiteral)
+        public override PatternUst Visit(PatternIntRangeLiteral patternIntRangeLiteral)
         {
-            if (patternIntLiteral.MinValue == patternIntLiteral.MaxValue)
+            if (patternIntRangeLiteral.MinValue == patternIntRangeLiteral.MaxValue)
             {
                 return new PatternIntLiteral(
-                    patternIntLiteral.MinValue,
-                    patternIntLiteral.TextSpan);
+                    patternIntRangeLiteral.MinValue,
+                    patternIntRangeLiteral.TextSpan);
             }
 
             return new PatternIntRangeLiteral(
```

Comment:
Why can't we just return `patternIntRangeLiteral` itself here? Why new instance which is a copy of the given one?